assets/src/apps/authoring/components/PropertyEditor/PropertyEditor.tsx

```diff
@@ -71,71 +76,96 @@ const PropertyEditor: React.FC<PropertyEditorProps> = ({
     setFormData(value);
   }, [value]);
 
-  return (
-    <Form
```

Comment:
These are formatting changes showing up because we wrapped them around a `<div>`. Changes are only in the `onChange` handler